src/test/api/karate/cards/patchCard.feature

```diff
@@ -0,0 +1,130 @@
+Feature: Cards
+
+
+  Background:
+
+    * def signIn = callonce read('../common/getToken.feature') { username: 'operator1_fr'}
+    * def authToken = signIn.authToken
+    * def signInAdmin = callonce read('../common/getToken.feature') { username: 'admin'}
+    * def authTokenAdmin = signInAdmin.authToken
+    * def perimeter =
+"""
+{
+  "id" : "perimeter",
+  "process" : "api_test",
+  "stateRights" : [
+      {
+        "state" : "messageState",
+        "right" : "ReceiveAndWrite"
+      }
+    ]
+}
+"""
+
+    * def perimeterArray =
+"""
+[   "perimeter"
+]
+"""
+
+  Scenario: Post card, patch card and then delete it.
+
+    * def card =
+"""
+{
+	"publisher" : "operator1_fr",
+	"processVersion" : "1",
+	"process": "api_test",
+	"processInstanceId" : "processForTestingPatch",
+	"state": "messageState",
+	"groupRecipients": ["Dispatcher"],
+	"severity" : "INFORMATION",
+	"startDate" : 1553186770681,
+	"summary" : {"key" : "defaultProcess.summary"},
+	"title" : {"key" : "defaultProcess.title"},
+	"data" : {"message":"a message"}
+}
+"""
+
+    * def cardPatch =
+"""
+{
+	"severity" : "COMPLIANT"
+}
```

Comment:
Patch data field + entityRecipients as well to have a more complete test

Check for example startDate , title and publisher has not been changed

Reply:
Done.